task/fbc-fips-check-oci-ta/0.1/fbc-fips-check-oci-ta.yaml

```diff
@@ -75,7 +75,7 @@ spec:
 
         echo "Inspecting raw image manifest $image_and_digest."
         # Get the arch and image manifests by inspecting the image. This is mainly for identifying image indexes
-        image_manifests=$(get_image_manifests -i "${image_and_digest}")
+        image_manifests=$(get_image_manifests -i ${image_and_digest})
```

Comment:
Why does this change remove the quotes around `${imageanddigest}`, while the changes in the other files adds the quotes?

Reply:
Thanks for finding. It is operated wrongly.
Because I found get_image_manifests has had `retry` when connecting to quay.io, so I wanted to undo my change on get_image_manifests.